code/modules/client/preference/loadout/loadout_general.dm

```diff
@@ -18,22 +18,91 @@
 	display_name = "a pack of Midoris"
 	path = /obj/item/storage/fancy/cigarettes/cigpack_midori
 
+/datum/gear/marijuanapack
```

Comment:
Not a fan of this for multiple reasons.

Reply:
def looks like marijuana packs aren't going in 😦 . Though to be fair iirc this PR was done before marijuana was given more healing properties